types/setting.go

```diff
@@ -255,6 +258,18 @@ var (
 		Default:     "300",
 	}
 
+	SettingDefinitionFailedBackupTTL = SettingDefinition{
+		DisplayName: "Failed Backup Time to Live",
+		Description: "In minutes. This setting determines how long Longhorn will keep the backup resource that was failed. Set to 0 to disable the auto-deletion.\n" +
+			"Failed backups will be checked and cleaned up during backupstore polling which is controlled by **Backupstore Poll Interval** setting. " +
```

Comment:
Hence this value determines the minimal wait interval of the cleanup. And the actual cleanup interval is multiple of **Backupstore Poll Interval**

Reply:
Yes, it is.

Reply:
suggest to add this sentence into description, right?

Reply:
Yes

Reply:
Added. Thanks.